inquire/src/lib.rs

```diff
@@ -28,7 +28,7 @@
 //!   - Custom extensions for files created by [`Editor`] prompts;
 //!   - and many others!
 //!
-//! \* Date-related features are available by enabling the `date` feature.
+//! \* Date-related features are available by enabling the `date` feature and enable the `editor` feature for the text editor features
```

Comment:
```suggestion
//! * The Editor and DateSelect prompts are available by enabling the `editor` and `date` features, respectively.
```

I think this reads a little better now that we're mentioning both prompts, wdyt?

Reply:
I agree